### PR TITLE
Javascript javascript2 week2

### PR DESCRIPTION
**Pull request by Lokesh-sw**

### CODE REVIEW

File: javascript/javascript2/week2/movies-excercise.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+console.log(longMovieTitle);
+
+//Count the number of movies made between 1980-1989 (including both the years)
+
+const movieYear = movies.filter(
+  (year) => year.year >= 1980 && year.year <= 1989
+);
+console.log(movieYear);
+
```
Comment (Vahablotfi) [inline javascript/javascript2/week2/movies-excercise.js:19]: The exercise asked for number of the movies.

**Review by Vahablotfi — COMMENTED** [review]
well done, you have a very good understanding of JavaScript higher order functions. 
I just think you need to make sure that the `movies-exercise.js` works fine before merging. It throw an error in console now.
I recommend that when you use array methods , pay attention to naming for example, you could use `movies.filter ((movie)=> ...)` or products.map((product)=> ...).

File: javascript/javascript2/week2/movies-excercise.js (hunk 1 above)
Comment (Lokesh-sw) [reply javascript/javascript2/week2/movies-excercise.js:19]: Added .length to reduce it to just numbers.

**Comment by Lokesh-sw** [comment]
> well done, you have a very good understanding of JavaScript higher order functions. I just think you need to make sure that the `movies-exercise.js` works fine before merging. It throw an error in console now. I recommend that when you use array methods , pay attention to naming for example, you could use `movies.filter ((movie)=> ...)` or products.map((product)=> ...).

**Comment by Lokesh-sw** [comment]
> > well done, you have a very good understanding of JavaScript higher order functions. I just think you need to make sure that the `movies-exercise.js` works fine before merging. It throw an error in console now. I recommend that when you use array methods , pay attention to naming for example, you could use `movies.filter ((movie)=> ...)` or products.map((product)=> ...).

"there was a little error in html regarding orders of links to js files, which sorted and now there is no error"

**Review by shravanipagilla — COMMENTED** [review]
✅ All tasks covered
✅ Application running fine
✅ Quality code
✅ wee done Lokesh

**Pull request merged**